Snowflake.Data.Tests/Snowflake.Data.Tests.csproj

```diff
@@ -14,6 +14,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="JunitXml.TestLogger" Version="3.0.134" />

```

Comment:
1. it's MIT license but do we have official aproval to use this lib?
2. I see that current version is 3.1.12 - can we use it?

Reply:
We need to go through portal to have it officially approved @sfc-gh-akolodziejczyk

Reply:
@sfc-gh-mhofman please review: https://github.com/snowflakedb/security_reviews/pull/4540

Reply:
I changed version to 3.1.12